Sync/SyncTelemetry.swift

```diff
@@ -148,19 +192,50 @@ extension SyncOperationStatsSession: DictionaryRepresentable {
 public struct SyncPing: TelemetryPing {
     public var payload: JSON
 
-    public init(opStats: SyncOperationStatsSession, engineStats: [SyncEngineStatsSession]) {
+    public init(account: FirefoxAccount?, syncOperationResult: SyncOperationResult) {
         var ping: [String: Any] = [
             "version": 1,
-            "discarded": 0,
             "why": SyncPingReason.schedule.rawValue,
-            "uid": "testUID",
-            "deviceID": "testDeviceID"
+            "uid": account?.uid ?? String(repeating: "0", count: 32)
```

Comment:
Just hard-code `"00000000000000000000000000000000"` if you think this is better than making it an optional field! Less generated code.